spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/lock/DefaultLockRepository.java

```diff
@@ -57,7 +57,7 @@
 	 */
 	public static final int DEFAULT_TTL = 10000;
 
-	private final String id = UUID.randomUUID().toString();
+	private String id = UUID.randomUUID().toString();
```

Comment:
No, this still has to be `final` and the default value must be populated from the default ctor.